pycheribuild/projects/syzkaller.py

```diff
@@ -62,92 +69,135 @@ def setup_config_options(cls, **kwargs):
             "run-sysgen", show_help=True,
             help="Rerun syz-extract and syz-sysgen to rebuild generated Go "
                  "syscall descriptions.")
+        cls.use_go = cls.add_config_option("use-go", kind=GoType, default=GoType.DEFAULT,
+                                           enum_choice_strings=[t.value for t in GoType],
+                                           help="The Go type to run with.")
+
+    def get_path(self):
+        return (str(self.get_sdk_bindir()) + ":" +
+                str(self.config.dollar_path_with_other_tools))
+
+    def get_sdk_dir(self, config):
```

Comment:
The config is always the one from the current object, we probably can just use `self.config here`